src/theme/_variables.css

```diff
@@ -45,8 +46,16 @@
 
   /* SHADOWS */
   --ck-color-shadow-drop: rgba(23, 73, 77, .15); /* CHROME-600 */
-  --ck-inner-shadow: var(--ck-color-focus-outer-shadow) 0 0 0 $ZD-SHADOWWIDTHS-MD inset;
   --ck-drop-shadow: var(--ck-color-shadow-drop) 0 20px 30px 0;
+  --ck-inner-shadow:
+    inset var(--ck-color-base-background) 0 0 0 $ZD-SHADOWWIDTHS-XS,
+    inset var(--ck-color-focus-border) 0 0 0 $ZD-SHADOWWIDTHS-MD;
+  --ck-focus-outer-shadow:
+    var(--ck-color-base-background) 0 0 0 $ZD-SHADOWWIDTHS-XS,
+    var(--ck-color-focus-outer-shadow) 0 0 0 $ZD-SHADOWWIDTHS-SM;
+  --ck-focus-disabled-outer-shadow:
+    var(--ck-color-base-background) 0 0 0 $ZD-SHADOWWIDTHS-XS,
+    var(--ck-color-focus-disabled-shadow) 0 0 0 $ZD-SHADOWWIDTHS-SM;
```

Comment:
Why was the total shadow width changed from the previous 3px? The focus for toolbar buttons is now inconsistent with Garden icon buttons.

Reply:
> so there is no overlap of focus over neighboring buttons

@lucijanblagonic the overlap here is intended by the original designs; it was deemed more important for these buttons to mimic Garden icon buttons after which they are patterned. I feel like that intent still holds.

Reply:
@jzempel I am OK with the decision to revert if we want to aim at consistency and we don't think overlap is much of an issue.

Reply:
updated in [cc62a90](https://github.com/zendeskgarden/ckeditor/pull/90/commits/cc62a901fe50cc2d3ace7e842cf66a09163b8fbb)!